terraform/dns-zone/01-variables.tf

```diff
@@ -0,0 +1,18 @@
+
+
```

Comment:
delete unnecessary empty lines at the beginning of files (here and in other files)